jetstream/cli.py

```diff
@@ -74,15 +74,13 @@ class ExecutorStrategy(Protocol):
 
     project_id: str
 
-    def __init__(self, project_id: str, dataset_id: str, *args, **kwargs) -> None:
-        ...
+    def __init__(self, project_id: str, dataset_id: str, *args, **kwargs) -> None: ...
```

Comment:
Yeah there were a lot of changes resulting from applying `black` here. I'm not sure if the version of `black` I was using is somehow different than the usual, because it generated a lot of changes to files I didn't touch.

Reply:
It looks like this specific change was added to `black` somewhat recently, so it must just be from updating the dependency. Let's just not worry about it for now (forgot that `black` doesn't let you ignore rules) since it's not that big of a deal. If this or any other `black` changes are bothersome enough we can consider moving to `ruff` or something instead.